package.json

```diff
@@ -2,7 +2,7 @@
 	"scripts": {
 		"npmlockrefresh": "npm i --package-lock-only",
 		"cm": "git-cz",
-	    "postinstall": "(cd device/samples && npm install && cd ../../) && (cd device/samples/pnp && npm install)"		
+	    "postinstall": "(cd device/samples/javascript && npm install && cd ../../) && (cd device/samples/javascript/pnp && npm install)"		
```

Comment:
do you think we should do npm install in both the javascript and typescript folders?

Reply:
I am okay with this

Reply:
we do the same thing with the pnp folders